src/lib.rs

```diff
@@ -878,7 +878,8 @@ struct Element<T> {
     next: MaybeIndex,
 }
 
-/// Index of coordinates in a cuboidal `grid_size.pow(DIM)` grid
+/// Index of coordinates in a `DIM`-dimensional grid. Each dimension is subdivided by `grid_size`,
+/// so the grid contains `grid_size.pow(DIM)` cells in total.
```

Comment:
The intent here is to describe a specific geometry, which "grid" doesn't accomplish. Perhaps "Index of coordinates in a grid of `DIM`-cubes" or similar?

Reply:
`DIM`-cubes sounds good.